data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh.template

```diff
@@ -10,12 +10,7 @@ until oc get baremetalhosts -n openshift-machine-api; do
    sleep 20
 done
 
-N="0"
-while [ "$N" -eq "0" ] ; do
-    echo "Waiting for control-plane bmh to appear..."
-    sleep 20
-    N=$(oc get bmh -n openshift-machine-api -l installer.openshift.io/role=control-plane --no-headers=true | wc -l)
-done
+N="{{ .PlatformData.BareMetal.MasterHostCount }}"
```

Comment:
`.sh.template` files don't get shellcheck run on them. It's better to pass this as an environment variable in the [systemd service](https://github.com/openshift/installer/blob/master/data/data/bootstrap/baremetal/systemd/units/master-bmh-update.service) so that we don't lose linting on shell scripts.